example/Vagrantfile

```diff
@@ -5,100 +5,134 @@
+            s.inline = "cd /home/vagrant/src/#{repo} && pip install -r requirements.txt"		
+            s.privileged = true
+            end
+          end
+        end
+        
         # Specify optional interface to bridge when wanting to expose the instance to an
         # external network.  Useful for when discovering nodes externally.
         if ENV['BRIDGE_INTF']
-           target.vm.network "public_network", ip: "172.31.128.1", bridge: "#{ENV['BRIDGE_INTF']}"
+          target.vm.network "public_network", ip: "172.31.128.1", bridge: "em1"
         else
-           target.vm.network "private_network", ip: "172.31.128.1", virtualbox__intnet: "closednet"
+          target.vm.network "private_network", ip: "172.31.128.1", virtualbox__intnet: "closednet"
         end
         target.vm.network "forwarded_port", guest: 8080, host: 9090
         target.vm.network "forwarded_port", guest: 5672, host: 9091
         target.vm.network "forwarded_port", guest: 9080, host: 9092
         target.vm.network "forwarded_port", guest: 8443, host: 9093
-
-        # NOTE: virtualbox shared folder syncing can be quite slow, but shared folders
-        # can be configured as NFS mounts to the host instead.
-        # To do so, append: `, type: "nfs"` to all of the target.vm.synced_folder commands
-        # below, then on the host system, run `sudo nfsd start`.
-        # Additionally, uncomment the below configuration:
-        #
-        # For NFS mounts, a host only network is needed. Feel free to change the IP.
-        # target.vm.network "private_network", ip: "192.168.50.4"
-
+        #usc-service
+        target.vm.network "forwarded_port", guest: 7080, host: 7080
+        
         # If true, then any SSH connections made will enable agent forwarding.
         # Default value: false
         target.ssh.forward_agent = true
 
         target.vm.provision "shell", inline: <<-SHELL
-          sudo service isc-dhcp-server start
-        #   service rsyslog stop
-        #   echo manual | sudo tee /etc/init/rsyslog.override
+          timeout=0
+          maxto=30
+          waitForPM2Daemon() {
+            while [ ${timeout} != ${maxto} ]; do
+              ps aux | grep PM2 | grep Daemon
+              if [ $? = 0 ]; then 
+                break
+              fi
+              sleep 1
+              timeout=`expr ${timeout} + 1`
+            done
+            if [ ${timeout} == ${maxto} ]; then
+              echo "Timed out waiting for PM2 Daemon (duration=`expr $maxto \* 1`s)."
+              exit 1
+            fi
+          }
+          service isc-dhcp-server start
+          service rsyslog stop
+          echo manual | sudo tee /etc/init/rsyslog.override
+          if [ "#{ENV['MULTI']}" ]; then
+            pm2 logs > /home/vagrant/src/build/vagrant.log &
+          else
+            pm2 logs > /home/vagrant/src/"#{ENV['REPO_NAME']}"/vagrant.log &
+          fi
+          waitForPM2Daemon
+          pm2 start rackhd-pm2-config.yml
```

Comment:
108-133 this whole sections seems to expect `pm2` to exist... since 2.0.0 I have not seen pm2 packaged with the example VM. I would suggest keeping the start of isc-dhcp, and removing the rest of this code.